docs/src/modules/java/pages/views.adoc

```diff
@@ -452,15 +456,116 @@ Use the following filter predicates to further refine results:
 * `<` less than
 * `\<=` less than or equals
 
-Combine filter conditions with the `AND` and `OR` operators.
+==== Logical operators
+
+Combine filter conditions with the `AND` and `OR` operators, and negate using the `NOT` operator. Group conditions using parentheses. Note that `AND` has precedence over `OR`.
 
 [source,proto,indent=0]
 ----
 SELECT * FROM customers WHERE
   name = :customer_name AND address.city = 'New York' OR
-  name = :customer_name AND address.city = 'San Francisco'
+  NOT (name = :customer_name AND address.city = 'San Francisco')
+----
+
+NOTE: `OR` support is currently disabled, until it can be more efficiently indexed.
+
+==== Array operators
+
+Use `IN` or `= ANY` to check whether a value is contained in a group of values or in an array column or parameter (a `repeated` field in the Protobuf message).
+
+Use `IN` with a list of values or parameters:
+
+----
+SELECT * FROM customers WHERE email IN ('bob@example.com', :some_email)
+----
+
+Use `= ANY` to check against an array column (a `repeated` field in the Protobuf message):
+
+----
+SELECT * FROM customers WHERE :some_email = ANY(emails)
+----
+
+Or use `= ANY` with a repeated field in the request parameters:
+
+----
+SELECT * FROM customers WHERE email = ANY(:some_emails)
+----
+
+==== Pattern matching
+
+Use `LIKE` to pattern match on strings. The standard SQL `LIKE` patterns are supported, with `_` (underscore) matching a single character, and `%` (percent sign) matching any sequence of zero or more characters.
+
+----
+SELECT * FROM customers WHERE name LIKE 'Bob%'
 ----
 
+NOTE: For index efficiency, only constant patterns with literal strings are supported, and the pattern must have a non-wildcard prefix or suffix. A pattern like `'%foo%'` is not supported.
```

Comment:
Do we need to clarify that request parameters can't be included in the pattern? You say that indirectly by "only constant patterns with literal strings" but maybe another addition at the end could make that even more clear?

Reply:
Ok, will massage.

Reply:
Added explicitly that request parameters can't be used.